BigBaseV2/src/views/view_self.cpp

```diff
@@ -31,6 +31,24 @@ namespace big
 			ImGui::Checkbox("Super Run", &g->self.super_run);
 			ImGui::Checkbox("No Idle Kick", &g->tunables.no_idle_kick);
 
+			ImGui::EndGroup();
+			ImGui::SameLine();
+			ImGui::BeginGroup();
+
+			ImGui::Checkbox("Invisibility", &g->self.invisibility);
+			ImGui::Checkbox("Keep Player Clean", &g->self.clean_player);
+			if (ImGui::Button("Clean Player"))
+			{
+				QUEUE_JOB_BEGIN_CLAUSE()
+				{
+					PED::CLEAR_PED_BLOOD_DAMAGE(PLAYER::PLAYER_PED_ID());
+					PED::CLEAR_PED_WETNESS(PLAYER::PLAYER_PED_ID());
+					PED::CLEAR_PED_ENV_DIRT(PLAYER::PLAYER_PED_ID());
+					PED::RESET_PED_VISIBLE_DAMAGE(PLAYER::PLAYER_PED_ID());
```

Comment:
Make these calls a function, you're using these same natives again in a looped feature.

util/entity.hpp
```cpp
inline void clean_ped(Ped ped)
{
	PED::CLEAR_PED_BLOOD_DAMAGE(PLAYER::PLAYER_PED_ID());
	PED::CLEAR_PED_WETNESS(PLAYER::PLAYER_PED_ID());
	PED::CLEAR_PED_ENV_DIRT(PLAYER::PLAYER_PED_ID());
	PED::RESET_PED_VISIBLE_DAMAGE(PLAYER::PLAYER_PED_ID());
}
```